src/generateRenderer/createSymbol.js

```diff
@@ -0,0 +1,26 @@
+const _ = require('lodash')
+
+module.exports = createSymbol
+
+function createSymbol (baseSymbol, color, type) {
+  const symbol = baseSymbol !== undefined ? _.cloneDeep(baseSymbol) : symbolTemplate(type) // TODO: figure out why cloneDeep is necessary
+  symbol.color = color
+  return symbol
+}
+
+function symbolTemplate (options) {
+  const renderers = {
```

Comment:
I had originally done this, but while debugging another issue, Ben suggested that it's good to have functions know exactly what they're dealing with... as in, if a renderer isn't used elsewhere, move it to that function.

But moving it out makes sense to me too.